third_party/npm-chalk/LICENSE

```diff
@@ -0,0 +1,9 @@
+MIT License
```

Comment:
Same as chromium, I don't think we should be duplicating any license files. Why aren't we just using special cases? https://github.com/brave/brave-core/pull/3899/files#diff-b7d947d83da72fb740477fc2782e945cR29

Reply:
None of the MIT licenses are the same. For example, this one includes this line:

> Copyright (c) Sindre Sorhus <sindresorhus@gmail.com> (sindresorhus.com)

It's not trivial to automate because we need to pull the license from GitHub and the naming is not consistent (`LICENSE`, `LICENCE`, `LICENSE.md`, `LICENSE.txt`, etc.). Also, sometimes it's missing (e.g. https://github.com/ericclemmons/unique-selector/pull/47).

Reply:
As a follow-up, I will look at creating a `src/brave/common-licenses/` directory so that we don't have to have more than one copy of the MPL or the GPL for example.